src/components/FeaturePanel/Properties/Direction.tsx

```diff
@@ -82,10 +186,10 @@ export const DirectionValue: React.FC<{ v: string }> = ({ children, v }) => {
         style={{
           display: 'flex',
           alignItems: 'center',
-          gap: '0.2rem',
+          gap: '0.5rem',
         }}
       >
-        <AnimatedArrow start={start} end={end ?? start} />
+        <DirectionIndicator start={start} end={end} />
```

Comment:
Could you perhaps make the line the same height as the other? Maybe make the indicator little smaller, and possibly with negative margins to overflow a little? 

![image](https://github.com/user-attachments/assets/74af3bf9-74fa-496c-8637-220939a6d07b)


Maybe centering the "Directions" label would be ok for start.